lib/handleRequest.js

```diff
@@ -3,6 +3,7 @@
 
 const fastJsonStringify = require('fast-json-stringify')
 const urlUtil = require('url')
+const errorUtil = require('./errorUtils')
```

Comment:
can you extract reflectSystemErrors?

Reply:
Do you mean `const reflectSystemErrors = require('./errorUtils').reflectSystemErrors` ?